app/models/catalog.rb

```diff
@@ -23,4 +24,9 @@ def set_title
       self.title = "#{season} #{year}"
     end
   end
+
+  private
+    def save_dimensions_now
+      image.analyze if image.attached?
```

Comment:
I think we shouldn't just analyze images if they are present because this can cause an unnecessary s3 call.  We need to only analyze if the image has not ben analyzed before.

Reply:
This method analyzes the image when it is added to the model. It should prevent any analysis when the image is subsequently called into a view via the imageable concern.

Reply:
This method is very resource intensive because of that s3 call. I'm pretty confident the reason .analyzed? was added as a method is to avoid using this method unnecessarily.